libs/design/guides/getting-started.md

```diff
@@ -103,5 +81,5 @@ With the module imported, the `AppComponent` template can be updated to render t
 </daff-hero>
 ```
 
-## Next Steps
-We've just walked through the basics of setting up the `@daffodil/design` library. There is much more to it than just the `DaffHeroComponent`. Check out the full list of available components, try and add them to your sample app, and imagine all the wonderful things you can now build!
\ No newline at end of file
+## Next steps
```

Comment:
Maybe leave a small challenge here? Something like "try to change the color of the hero" and link to the docs of the hero component?

Reply:
added